net_utils.js

```diff
@@ -421,3 +428,32 @@ exports.get_ipany_re = function (prefix, suffix, modifier) {
         modifier
     );
 };
+
+exports.ipv6_reverse = function(ipv6){
+    ipv6 = new IPv6.Address(ipv6);
```

Comment:
Missing 'var' declaration.

Additionally - do we really need the 'ip-address' module for this being as we already have the ipaddr.js module.